packages/card/index.tsx

```diff
@@ -1,3 +1,7 @@
 import Card from './card'
+import * as CardPrimitives from './primitives'
```

Comment:
This was done (as well as the export below) in order to use `CardPrimitives` in `docs.mdx` examples.